universum/submit.py

```diff
@@ -22,29 +22,44 @@ def define_arguments(parser):
                             help="Only submit existing vcs modifications, no adding or deleting")
 
         parser.add_argument('--commit-message', '-cm', dest='commit_message', metavar="COMMIT_MESSAGE",
-                            help='Commit message to add')
-        parser.add_argument("--reconcile-list", "-rl", action="append", nargs='+', dest="reconcile_list",
-                            metavar="RECONCILE_LIST",
-                            help="List of vcs or directories to be reconciled for commit. "
-                                 "Relative paths starting at client root are supported")
+                            help="Change description. Enter in command line directly, store in environment variable, "
```

Comment:
`Change description`, description of .. ?
Probably something like `Change description of submitting changelist`

Reply:
Description of a _change_. In case of Git there are no changelists, there only is a commit and a commit message.
Both 'commit' and 'changelist' are changes to be submitted that need description. But I'll think of rephrasing if this one is misleading